incubation-openmetrics.md

```diff
@@ -0,0 +1,46 @@
+# OpenMetrics proposal for CNCF incubation
+
+## Background
+
+OpenMetrics specifies the de-facto standard for transmitting cloud-native metrics at scale, with support for both text representation and Protocol Buffers. It’s the wire format and data model Prometheus, Cortex, and Thanos rely on and are migrating more componentes to.
+
+At the same time, OpenMetrics is pursuing an RFC under IETF OPSAWG https://datatracker.ietf.org/doc/draft-richih-opsawg-openmetrics/ . 
+
+OpenMetrics was created outside of Prometheus at the request of Dan Kohn and Chris Aniszczyk in 2015 and named OpenMetrics in 2016.
+
+OpenMetrics came out 5th place in the CNCF End User Survey on Observability tools.
+
+OpenMetrics entered the CNCF Sandbox in July 2018.
+
+## Progress since joining the CNCF Sandbox
+
+OpenMetrics carefully evolved and tightened Prometheus exposition format 0.0.4.
+
+Here are some highlights:
+
+ * Prometheus has supported and used OpenMetrics on the wire since 2018
+ * Cortex and Thanos support it
+ * DataDog has reimplemented OpenMetrics on their own without help from OpenMetrics
+ * OpenMetrics has been submitted to IETF OPSAWG
+ * IETF OPSAWG showed very strong interest in quickly releasing OpenMetrics after a presentation at IETF 110
+ * Exemplar support has been merged into Prometheus main and will be part of the next release. This would not ever have happened without OpenMetrics. This directly enables users to correlate metrics with traces as cloud-native scale.
+
+### Transition from Prometheus exposition format 0.0.4 to OpenMetrics 1.0
+
+This is handled transparently by the Prometheus client libraries.
+
+There are two changes, one breaking:
+
+ * Timestamps are in seconds, not microseconds. This was the only place where Prometheus used microseconds on the wire. As timestamps are not recommended in Prometheus expositions anyway and largely considered a hack/workaround for very few and specific situations, users are very unlikely to hit this.
```

Comment:
s/micro/milli/g

Also, they'd have to expose the wrong content type for this to be an issue.

Reply:
Good points, will edit.